tests/integration/test_agent.py

```diff
@@ -218,16 +231,13 @@ def test_ipython_module(current_test_name: str):
 
 
 @pytest.mark.skipif(
-    os.getenv('DEFAULT_AGENT') != 'BrowsingAgent'
-    and os.getenv('DEFAULT_AGENT') != 'CodeActAgent',
-    reason='currently only BrowsingAgent and CodeActAgent are capable of searching the internet',
+    os.getenv('DEFAULT_AGENT') != 'BrowsingAgent',
+    reason='currently only BrowsingAgent is capable of searching the internet',
 )
 @pytest.mark.skipif(
-    (
-        os.getenv('DEFAULT_AGENT') == 'CodeActAgent'
-        or os.getenv('DEFAULT_AGENT') == 'CodeActSWEAgent'
-    ),
-    reason='CodeActAgent/CodeActSWEAgent only supports ssh sandbox which is stateful',
+    os.getenv('DEFAULT_AGENT') == 'CodeActSWEAgent'
+    or os.getenv('DEFAULT_AGENT') == 'CodeActAgent',
+    reason='CodeActAgent/CodeActSWEAgent do not support browsing',
```

Comment:
Why? `CodeActAgent` should support browsing

Reply:
Right, will revert that, was trying it out for testing.